demosplan/DemosPlanCoreBundle/Logic/Segment/SegmentsExporter.php

```diff
@@ -92,6 +92,28 @@ protected function addHeader(Section $section, Procedure $procedure): void
         );
     }
 
+    public function getSimilarStatementSubmitters(Statement $statement)
+    {
+        $submitterStrings = [];
+        foreach ($statement->getSimilarStatementSubmitters() as $submitter) {
+            $values = [
+                $submitter->getEmailAddress(),
+                $submitter->getStreetNameWithStreetNumber(),
+                $submitter->getPostalCodeWithCity(),
+            ];
+            $values = array_filter($values, fn (?string $value): bool =>null !== $value);
```

Comment:
Just a minor improvement.

```suggestion
            $values = array_filter($values, static fn (?string $value): bool => null !== $value);
```